Sources/SecureEnclaveValet.swift

```diff
@@ -90,8 +90,8 @@ public final class SecureEnclaveValet: NSObject {
     
     // MARK: Hashable
     
-    public override var hashValue: Int {
-        return service.description.hashValue
+    public override var hash: Int {
```

Comment:
Can we write this as
```
#if swift(>=4.2)
public override var hash: Int {
    return service.description.hash
}
#else
public override var hashValue: Int {
    return service.description.hashValue
}
#endif
```

Here and below? That way I can release a version with this sha without breaking Xcode 9 folks.

Reply:
I thought about that at first, too, but this approach still produces the warning in Xcode 10. It's not that `hash` is a new API; rather, `hashValue` comes from the `Hashable` protocol and `hash` comes from `NSObject`; since these classes inherit from `NSObject` the guidance from Xcode is to use `hash` instead. I did test that this code works on Xcode 9.3.

Reply:
Got it! Appreciate the explanation.

Your explanation helped me find some supportive documentation in [Swift's CHANGELOG](https://github.com/apple/swift/blob/master/CHANGELOG.md) re using `hash` in `NSObject` subclasses. Under the subheading of [SE-0206](https://github.com/apple/swift-evolution/blob/master/proposals/0206-hashable-enhancements.md) under Swift 4.2, the log mentions that the new `hashValue`/`Hashable` changes `don't affect Foundation's hashing interface. Classes that subclass NSObject should override the hash property, like before.`

With that, your PR LGTM. Will merge.

Reply:
Thanks for finding that. I was trying to find some actual concrete documentation on `hash` vs. `hashValue` and this looks like the closest thing.